inference-engine/src/transformations/src/transformations/common_optimizations/weights_dequantize_to_fake_quantize.cpp

```diff
@@ -62,8 +62,8 @@ ngraph::pass::WeightsDequantizeToFakeQuantize::WeightsDequantizeToFakeQuantize()
         ngraph::copy_runtime_info(nodes_to_copy_RT_info_from, fq);
         multiply_node->output(0).replace(fq->output(0));
 
-        if (convert_node->get_rt_info().count("DISABLED_CONSTANT_FOLDING"))
-            convert_node->get_rt_info().erase("DISABLED_CONSTANT_FOLDING");
+        if (convert_node->get_rt_info().count("disabled_constant_folding_0"))
```

Comment:
should there be helper `ov::has_disabled_constant_folding`/ `ov::get_disabled_constant_folding` like there is `ov::disable_constant_folding` which hides logic with "_0"?
In future when new version is introduced, will we have to add `node->get_rt_info().erase("disabled_constant_folding_1")`?

Reply:
There are some components like ngraph library which doesn't depend on transformation library but still check this attribute. So for them this approach wont' work. For this particular case this check is rather a WA which needs to be removed.

Reply:
Changed for CommonOptimizations. In future we will move this attributes directly to NG so we can update NG facilities.